src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 32px 19px 36px;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
```

Comment:
Be careful with setting both width and height on text containers, as this can lead to text clipping if the content exceeds the fixed size. Consider using min-height instead, or padding to ensure the content fits.